packages/visualizations-react/package.json

```diff
@@ -47,31 +53,60 @@
     },
     "dependencies": {
         "@opendatasoft/visualizations": "^0.9.0",
+        "@testing-library/jest-dom": "^5.11.9",
+        "@testing-library/react": "^11.2.5",
+        "@types/jest": "^29.0.0",
         "use-callback-ref": "^1.2.4"
     },
     "devDependencies": {
-        "@babel/core": "^7.12.10",
-        "@size-limit/preset-small-lib": "^4.9.1",
+        "@babel/core": "^7.19.0",
+        "@babel/preset-env": "^7.19.0",
+        "@babel/preset-react": "^7.18.6",
+        "@babel/preset-typescript": "^7.18.6",
+        "@rollup/plugin-babel": "^5.3.0",
+        "@rollup/plugin-commonjs": "^19.0.0",
+        "@rollup/plugin-json": "^4.1.0",
+        "@rollup/plugin-node-resolve": "^10.0.0",
+        "@rollup/plugin-replace": "^3.0.1",
+        "@rollup/plugin-typescript": "^6.1.0",
+        "@size-limit/preset-small-lib": "^8.1.0",
         "@storybook/addon-essentials": "^6.4.22",
-        "@storybook/addon-info": "^5.3.21",
+        "@storybook/addon-info": "^4.1.18",
         "@storybook/addon-links": "^6.4.22",
         "@storybook/addons": "^6.4.22",
-        "@storybook/react": "^6.4.22",
-        "@testing-library/jest-dom": "^5.11.9",
-        "@testing-library/react": "^11.2.5",
-        "@types/jest": "^26.0.20",
+        "@storybook/react": "^6.1.21",
         "@types/react": "^17.0.0",
         "@types/react-dom": "^17.0.0",
+        "@typescript-eslint/eslint-plugin": "^5.37.0",
+        "@typescript-eslint/parser": "^5.37.0",
         "babel-loader": "^8.2.2",
         "chromatic": "^5.7.1",
+        "eslint": "^7.32.0",
+        "eslint-config-airbnb": "^19.0.4",
+        "eslint-config-airbnb-typescript": "^17.0.0",
+        "eslint-config-prettier": "^8.3.0",
+        "eslint-plugin-import": "^2.26.0",
+        "eslint-plugin-jsx-a11y": "^6.6.1",
+        "eslint-plugin-no-only-tests": "^3.0.0",
+        "eslint-plugin-prettier": "^4.2.1",
+        "eslint-plugin-react": "^7.31.8",
+        "eslint-plugin-react-hooks": "^4.6.0",
         "husky": "^4.3.7",
+        "jest": "^29.0.2",
         "jest-canvas-mock": "^2.3.0",
+        "jest-environment-jsdom": "^29.0.2",
+        "jest-fetch-mock": "^3.0.3",
+        "npm-run-all": "^4.1.5",
+        "postcss": "^8.4.16",
         "react": "^16.13.0",
         "react-dom": "^16.13.0",
         "react-is": "^16.13.0",
-        "size-limit": "^4.9.1",
-        "tsdx": "^0.14.1",
+        "rollup": "^2.33.1",
```

Comment:
Was it the version used by TSDX?

It is 2 years old now, I agree we should stick to the TSDX one in this PR, but in the near future, bumping rollup across our 3 packages could be nice. I suppose there were a loooot of bugfixes and improvements. (I'm not sure how many breaking changes though 😅)

Reply:
Actually I'm of the opinion of starting with an up-to-date rollup since it doesn't seem to break anything to upgrade 👌